test/common/json/config_schemas_test_data/BUILD

```diff
@@ -0,0 +1,11 @@
+package(default_visibility = ["//visibility:public"])
+
+load("//bazel:envoy_build_system.bzl", "envoy_py_test_binary")
+
+envoy_py_test_binary(
+    name = "generate_test_data",
+    srcs = [
```

Comment:
I'm not positive, but I'm guessing that unless you export the other python files in the directory, the actual schema tests in the other files won't run. We might want to put some expected number of loaded test files in the test main loop to catch this (like I recently did for the example config tests).